### PR TITLE
Fix NDEV-15015

### PR DESCRIPTION
**Pull request by atulatnirmata**
- Updated the README for base, nirmata and kyverno-operator
- Embedded the instructions for installing kyverno-operator as a prerequisite for n4k
- Updated the instructions with the username=nirmata-enterprise-for-kyverno instead of someuser
- Updated some formatting

### CODE REVIEW

File: charts/kyverno-operator/README.md [hunk 1]
```diff
@@ -6,50 +6,76 @@
...
+```console
 helm repo add nirmata https://nirmata.github.io/kyverno-charts/
+helm repo update nirmata
+```
 
-# 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+### 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+Create the namespace
+```console
+kubectl create namespace nirmata-kyverno-operator
```
Comment (aniket-nirmata) [inline charts/kyverno-operator/README.md:24]: We can remove this step and add `--create-namespace` in the following command

File: charts/nirmata/README.md [hunk 2]
```diff
@@ -8,25 +8,68 @@
...
+#### 1. Add Kyverno Helm Repository
+```console
+helm repo add nirmata https://nirmata.github.io/kyverno-charts/
+helm repo update nirmata
+```
+
+#### 2. (Optional) If a custom CA is used, create a configmap corresponding to the same with key custom-ca.pem. E.g.
+Create the namespace
+```console
+kubectl create namespace nirmata-kyverno-operator
```
Comment (aniket-nirmata) [inline charts/nirmata/README.md:32]: We can remove this step and add --create-namespace in the following install command if not present

**Review by patelrit — APPROVED** [review]

File: charts/kyverno-operator/README.md (hunk 1 above)
Comment (atulatnirmata) [reply charts/kyverno-operator/README.md:24]: kubectl does not have a --create-namespace option. Does it?

Comment (aniket-nirmata) [reply charts/kyverno-operator/README.md:24]: @atulatnirmata I meant to put this in `helm install ` command

Comment (atulatnirmata) [reply charts/kyverno-operator/README.md:24]: The namespace is created for the configmap, which has to be done before the helm install.
The helm install already has `--create-namespace`

Comment (aniket-nirmata) [reply charts/kyverno-operator/README.md:24]: Ohh, Okay. No need to merge.

**Review by aniket-nirmata — APPROVED** [review]

**Pull request merged**